### PR TITLE
Added plugin manager

### PR DESCRIPTION
**Pull request by danielepantaleone**
This PR implements #143 (what is described by @ozon in the issue body, without the download/install enhancements).

Current available commands:

* **!plugin list** - list loaded plugins
* **!plugin enable foo** - enable a plugin
* **!plugin disable foo** - disable a plugin
* **!plugin load foo** - load a plugin (no need for B3 reboot)
* **!plugin unload foo** - unload a loaded plugin
* **!plugin info foo** - display plugin information (author and version)

### CODE REVIEW

File: b3/plugins/pluginmanager/__init__.py [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+            client.message('^7Plugin ^1%s ^7is protected' % name)
+        else:
+            plugin = self.console.getPlugin(name)
+            if not plugin:
+                client.message('^7Plugin ^1%s ^7is not loaded' % name)
+            else:
+                if plugin.isEnabled():
+                    client.message('^7Plugin ^1%s ^7is currently enabled: disable it first' % name)
+                else:
+                    del self.console._plugins[name]
```
Comment (thomasleveil) [inline b3/plugins/pluginmanager/__init__.py]: what about the commands, events handler and cron tasks that plugin registered?

Comment (danielepantaleone) [inline b3/plugins/pluginmanager/__init__.py]: I knew I forgot something...I'll fix it tomorrow

Inviato da iPhone

**Comment by danielepantaleone** [comment]
It should work properly now

**Comment by thomasleveil** [comment]
nice work :+1: LGTM

**Pull request merged**